### ISSUE TITLE
rmonitor does not work with Condor and output in directories

### ISSUE THREAD

**Issue by btovar** [cooperative-computing-lab/cctools#1364]
We rely on Condor to bring back files in the task scratch directory. This does not work when the resource_monitor writes its output to a directory.

**Comment by btovar** [comment]
Some options:

- Specify transfer_output_files in Condor, and add a periodic remove so that jobs are never held when output is missing.

- The monitor wrapper tells the monitor to always write in the task top-most directory. When the task finishes, the wrapper (locally) moves the output to the desired destination. This is what we do in Work Queue, but with remote renaming.

**Comment by btovar** [comment]
@nhazekam: The monitor works correctly on -Tsge. Please check with slurm on xsede.

**Comment by dthain** [comment]
IIRC, Condor can be induced to retrieve a directory of output, if it is specified in the submit file.

**Comment by btovar** [comment]
Yes, but we do not specify any of the outputs in condor. This is so that if the task fails, the condor job is not held. We could change that, though, telling condor never to hold a job on missing output.
